test/Options/SimplePutOption.js

```diff
@@ -20,45 +20,278 @@ const { ethers } = require("hardhat");
  * TODO: Complete test's use Integration.js for inspiration
  */
 describe("Simple Put Option Test Suite", function () {
-  let mockAggregator;
-  this.beforeEach(async function () {
-    // mockAggregator = await ethers.getContractFactory(MockAggregatorV3Interface);
-    // mockAggregator = await mockAggregator.deploy(18, 5000000000000000);
-    // await mockAggregator.deployed();
-    // const mockedAggregatorV3Interface = await loadFixture(
-    //   async function deployMockedAggregatorV3Interface() {
-    //     const [owner, otherAccount] = await ethers.getSigners();
-    //     const AggregatorV3Interface = await ethers.getContractFactory(
-    //       "AggregatorV3Interface"
-    //     );
-    //     const aggregatorV3Interface = await AggregatorV3Interface.deploy();
-    //     return { aggregatorV3Interface, owner, otherAccount };
-    //   }
-    // );
-    // const mockedPriceReferenceLevel = await loadFixture(
-    //   async function deployMockedPriceReferenceLevel() {
-    //     const [owner, otherAccount] = await ethers.getSigners();
-    //     const MockedPriceReferenceLevel = await ethers.getContractFactory(
-    //       "MockedPriceReferenceLevel"
-    //     );
-    //     const priceReferenceLevel = await MockedPriceReferenceLevel.deploy(
-    //       "MockedPriceTicker",
-    //       mockAggregator.address
-    //     );
-    //     return { priceReferenceLevel, owner, otherAccount };
-    //   }
-    // );
+  // Define accounts
```

Comment:
Same comments as for SimpleOption.js - re: formatting, and refactor of shared functions